instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/jaeger/JaegerSpanExporterProperties.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.exporters.jaeger;
+
+import java.time.Duration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration for {@link JaegerSpanExporter}
+ *
+ * <p>Get Exporter Service Name {@link getServiceName()}
+ *
+ * <p>Get Exporter Host Name {@link getHost()}
+ *
+ * <p>Get Exporter Port {@link getPort()}
+ *
+ * <p>Get max wait time for Collector to process Span Batches {@link getDeadline()}
+ */
+@ConfigurationProperties(prefix = "opentelemetry.trace.exporter.jaeger")
+public final class JaegerSpanExporterProperties {
+
+  private boolean enabled = true;
```

Comment:
[Note] I'm not sure about the convention, but I'd consider making the defaults part of the class documentation. 

I'd also add a link to where that 14250 is coming from.